evalml/model_understanding/permutation_importance.py

```diff
@@ -293,6 +293,7 @@ def _shuffle_and_score_helper(
         col = X_permuted.iloc[shuffling_idx, col_idx]
         col.index = X_permuted.index
         X_permuted.iloc[:, col_idx] = col
+        X_permuted.ww.init(schema=X_features.ww.schema)
```

Comment:
Modifying the dataframe outside of Woodwork, as is being done with `X_permuted.iloc[:, col_idx] = col`, always carries the risk of invalidating the schema.

That said, I don't think it is currently possible to do this type of assignment through WW currently with this:
```
X_permuted.ww.iloc[:, col_idx] = col
```

I'm not sure how easy it would be to implement, but we could look into it if needed.

Reply:
By the way, based on my _limited_ testing, as long as the new values don't change the column `dtype`, the schema should not be invalidated by this type of assignment outside of WW.